includes/helper/class-wp-job-manager-helper-nonce.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * File containing the class WP_Job_Manager_Helper_Nonce.
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WP_Job_Manager_Helper_Nonce
+ */
+class WP_Job_Manager_Helper_Nonce {
```

Comment:
We now have a third class after `WP_Job_Manager_Site_Trust_Token` and `Access_Token` which roughly do the same thing. It would be a good idea to merge these at some point, but let's consider this as part of another PR.

Reply:
Hey there!

hmmm, I considered using the `Site_Trust_Token` thing because it is a bit similar, yeah, however, in this case we're looking for a *global* authorization, not really associated with a post or user or something similar. So, the Site Trust Token idea wouldn't work here.

That said, I just learned about `Access_Token`, and I can confirm that it should work perfectly! 

Good catch!